synfig-studio/src/gui/app.cpp

```diff
@@ -1964,7 +1964,7 @@ App::save_settings()
 		{
 			std::string filename=get_config_file("language");
 
-			std::ofstream file(filename.c_str());
+			std::ofstream file(FileSystemNative::path(filename).c_str());
```

Comment:
Since C++11, we don't need to use `.c_str()` in `std::ofstream` constructor

Reply:
@rodolforg 
Custom `std::ofstream(...)` works with `wchar_t*` argument, not with `std::wstring`, so here we need to convert it.

VS/MinGW:
```
explicit basic_ofstream(
    const char* _Filename,
    ios_base::openmode _Mode = ios_base::out,
    int _Prot = (int)ios_base::_Openprot);

explicit basic_ofstream(
    const wchar_t* _Filename,
    ios_base::openmode _Mode = ios_base::out,
    int _Prot = (int)ios_base::_Openprot);

```